src/probnum/diffeq/ode/__init__.py

```diff
@@ -1,6 +1,14 @@
-from .ivp import *
-from .ivp_examples import *
-from .ode import *
+from .ivp import IVP
+from .ivp_examples import (
```

Comment:
You do not import the `*_rhs()` methods from `ivp_examples`. I personally find that desirable, I just wanted to check whether that was a deliberate choice.

Reply:
This was a deliberate choice